src/bindings/python-2/configure.ac

```diff
@@ -0,0 +1,174 @@
+#                                               -*- Autoconf -*-
+# Process this file with autoconf to produce a configure script.
+
+AC_PREREQ([2.63])
+AC_INIT([flux-python-2],
+        m4_esyscmd([git describe --always | awk '/.*/ {sub(/^v/, ""); printf "%s",$1; exit}']))
+# AC_CONFIG_SRCDIR([NEWS])
+# AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_AUX_DIR([config])
+AC_CONFIG_MACRO_DIR([config])
+AM_INIT_AUTOMAKE([subdir-objects])
+AM_SILENT_RULES([yes])
+
+#LT_PREREQ([2.4.2])
+LT_INIT([dlopen])
+
+# Checks for programs.
+AC_DEFINE([_GNU_SOURCE], 1,
+          [Define _GNU_SOURCE so that we get all necessary prototypes])
+AC_PROG_CC_C99
+AC_PROG_LN_S
+AC_PROG_MAKE_SET
+AM_PROG_CC_C_O
+
+if test "$GCC" = yes; then
+  WARNING_CFLAGS="-Wall -Werror -Werror=missing-field-initializers -Wno-error=deprecated-declarations"
+  AC_SUBST([WARNING_CFLAGS])
+fi
+if test "$GXX" = "yes"; then
+  WARNING_CXXFLAGS=$WARNING_CFLAGS
+  AC_SUBST([WARNING_CXXFLAGS])
+fi
+
+# Checks for libraries.
+PKG_PROG_PKG_CONFIG
+
+#  Edit PATH to remove $PWD/src/cmd so that AM_PATH_PYTHON doesn't find
+#  flux python script (thus creating a link to itself.) This needs to be
+#  done *before* AX_PYTHON_DEVEL.
+#
+saved_PATH=$PATH
+export PATH=$(echo $PATH | sed "s|$(pwd)/src/cmd:*||")
+
+# avoid pollution from outer env vars
+PYTHON=$PYTHON2
+PYTHON_VERSION=$PYTHON2_VERSION
+AX_PYTHON_DEVEL([<'2.8'])
+
+AM_PATH_PYTHON([$ac_python_version])
+if test "X$PYTHON" = "X"; then
+  AC_MSG_ERROR([could not find python])
+fi
+if test "X$PYTHON" = "X"; then
+  AC_MSG_ERROR([could not find python])
+fi
+#  Restore original PATH:
+export PATH=${saved_PATH}
+
+# Flag for PYTHON_LDFLAGS workaround below.
+if test -n "$PYTHON_LDFLAGS"; then
+  ac_python_ldflags_set_by_user=true
+fi
+
+AM_CHECK_PYMOD(cffi,
+               [cffi.__version_info__ >= (1,1)],
+               ,
+               [AC_MSG_ERROR([could not find python module cffi, version 1.1+ required])]
+               )
+AM_CHECK_PYMOD(six,
+               [StrictVersion(six.__version__) >= StrictVersion('1.9.0')],
+               ,
+               [AC_MSG_ERROR([could not find python module six, version 1.9.0+ required])]
+               )
+AM_CHECK_PYMOD(yaml,
+               [StrictVersion(yaml.__version__) >= StrictVersion ('3.10.0')],
+               ,
+               [AC_MSG_ERROR([could not find python module yaml, version 3.10+ required])]
+               )
+AM_CHECK_PYMOD(jsonschema,
+               [StrictVersion(jsonschema.__version__) >= StrictVersion ('2.3.0')],
+               ,
+               [AC_MSG_ERROR([could not find python module jsonschema, version 2.3.0+ required])]
+               )
```

Comment:
I'm not 100% sure on this, but I think this dependency only exists for commands in `src/cmd` and `t/`, which I think will only be run under python3.  So we may be able to drop this dependency for the bindings generated against python2.

Reply:
Thanks for that, currently this runs all, the python tests from t/, I'll double-check that none of them need it and if not drop it.